packages/mock-app/src/__tests__/JsonBlock.test.tsx

```diff
@@ -34,29 +33,14 @@ describe('Json block content', () => {
     },
   };
 
+  const renderComponent = (verifiableCredential: any) => {
+    return render(<JsonBlock credential={credential} />);
+  };
```

Comment:
I'm a bit curious about this function.
Why did you wrap the `render` function from the '@testing-library/react' library inside this function, and why aren't you using the function's argument?

Reply:
Before, I wrote many test cases and needed to call the render many times. But after refactoring, only one test case now.